benchmarks/data/generate_data_multi_file.py

```
@@ -0,0 +1,516 @@
+    """
+    result = {}
+
+    n_rowgroups = 20
+    n_vectors_per_rowgroup = 64
+    n_vectors = n_rowgroups * n_vectors_per_rowgroup
+
+    for n_cols in [cols]:
+        # for n_vectors in range(
+        #     n_vectors_per_rowgroup, n_vectors + 1, n_vectors_per_rowgroup
+        # ):
+        for n_vectors in [n_vectors]:
```

Ruff (0.11.9)

Loop control variable `n_vectors` overrides iterable it iterates

(B020)